locksmith/src/websub/jobs/keys.ts

```diff
@@ -97,6 +116,284 @@ async function notifyHooksOfAllUnprocessedKeys(hooks: Hook[], network: number) {
   }
 }
 
+const getRecipient = async (lockAddress: string, ownerAddress: string) => {
```

Comment:
I think we are using this in multiple places? Maybe move to `userMetadataOperations` as well so it is more reusable?